zebra/zebra_fpm_netlink.c

```diff
@@ -274,9 +274,16 @@ static int netlink_route_info_fill(struct netlink_route_info *ri, int cmd,
 				   rib_dest_t *dest, struct route_entry *re)
 {
 	struct nexthop *nexthop;
-	struct rib_table_info *table_info =
-		rib_table_info(rib_dest_table(dest));
-	struct zebra_vrf *zvrf = table_info->zvrf;
+	struct rib_table_info *table_info;
+	struct zebra_vrf *zvrf;
+
+	if (!re) {
```

Comment:
No, looks like this will also break DELROUTE, which looked like it allowed a NULL `re` (since that path did not need nexthops)

Reply:
Please check the latest, thanks.